libs/src/services/discoveryProvider/index.js

```diff
@@ -144,6 +146,9 @@ class DiscoveryProvider {
     if (sort) {
       req.queryParams.sort = sort
     }
+    if (typeof filterDeleted === 'boolean') {
```

Comment:
Why the `typeof` check rather than just truthy check?

Reply:
In case we ever wanted to pass in a value of `false`